reference/conanfile/attributes.rst

```diff
@@ -1348,6 +1348,32 @@ Its basic syntax is:
 
 Read more about this attribute in :ref:`python_requires`
 
+
+python_requires_extend
+----------------------
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+
+This class attribute defines one or more classes that will be injected in runtime as base classes of
+the recipe class. Syntax for each of these classes should be a string like ``pyreq.MyConanfileBase``
```

Comment:
also, it's not obvious what's the effect of injection.
e.g. will `class Pkg(ConanFile):` transform into `class Pkg(pyreq.MyConanfileBase):`? or will it transform into `class Pkg(pyreq.MyConanfileBase, ConanFile)` (multiple inheritance)? or something else?

Reply:
I've added it to the `python_requires` section (the linked one)